yarn.lock

```diff
@@ -1960,11 +1960,6 @@ JSONStream@^1.0.4:
     jsonparse "^1.2.0"
```

Comment:
I also had differences in my yarn.lock file. I guess we forgot to commit it somewhere

Reply:
should I remove it from the pr, or leave it?

Reply:
Keep it in, hopefully this will fix things for everyone :)